db/migrate/20170922172113_rename_provisioning_manager_id_to_manager_id_in_customization_scripts.rb

```diff
@@ -0,0 +1,5 @@
+class RenameProvisioningManagerIdToManagerIdInCustomizationScripts < ActiveRecord::Migration[5.0]
+  def change
+    rename_column :customization_scripts, :provisioning_manager_id, :manager_id
```

Comment:
Would you mind splitting this into 2 PR's?  See https://github.com/ManageIQ/manageiq/pull/16036#discussion_r142487333